xdebug_com.c

```diff
@@ -25,165 +25,205 @@
 # include <sys/select.h>
 #endif
 #ifndef PHP_WIN32
-#include <unistd.h>
-#endif
-
-#ifdef PHP_WIN32
-# include <process.h>
-# include <direct.h>
-# include "win32/time.h"
-#define PATH_MAX MAX_PATH
-#else
+# include <unistd.h>
 # include <sys/socket.h>
-# include <netinet/in.h>
 # include <netinet/tcp.h>
-# include <arpa/inet.h>
 # include <netdb.h>
+#else
+# include <process.h>
+# include <direct.h>
+# include "win32/time.h"
+# undef UNICODE
+# include <winsock2.h>
+# include <ws2tcpip.h>
+# pragma comment (lib, "Ws2_32.lib")
+# define PATH_MAX MAX_PATH
 #endif
 
 #include "xdebug_com.h"
 
-#ifdef PHP_WIN32
```

Comment:
My implementation doesn't use the inet_aton function anymore, so I removed it. Is there any other code that needs it?